### ISSUE TITLE
Keyboard user must be able to close Bottom Sheet

### ISSUE THREAD

**Issue by marcysutton** [angular/material#729]
When a bottom sheet opens, the only way to exit is to click somewhere else on the screen because there is no close button. The escape key needs to be wired up to close the component, and focus should be sent back to the triggering item. A separate issue has been filed for focus management of all Interim Elements: https://github.com/angular/material/issues/728
